cypresse2e/cypress/integration/e2eTests/helpBoard/helpBoard.spec.js

```diff
@@ -21,6 +24,54 @@ describe("FightPandemics Help Board Page for unauthorized user", () => {
       cy.validateCorrectScreenIsOpen("auth/login");
     });
 
+    it("keyword search Icon is visible and clickable", () => {
+      var searchIcon = cy.get(helpBoard.getKeywordSearchButton());
+      searchIcon
+        .should("be.visible")
+        .and("have.attr", "alt", "Icon")
+        .click({ multiple: true, force: true });
+      //cy.keywordSearchIconIsVisibleAndClickable(helpBoard.keywordSearchButton);
+    });
```

Comment:
Since you already have the searchIcon function written below, you can just simply call `searchIcon().click({ multiple: true, force: true })` in this testcase.